internal/testdata/flink_statement/read_resumed_flink_statement.json

```diff
@@ -26,7 +26,7 @@
       "customers_source": "partition:0,offset:9223372036854775808;partition:4,offset:9223372036854775808;partition:3,offset:9223372036854775808;partition:2,offset:9223372036854775808;partition:1,offset:9223372036854775808;partition:5,offset:9223372036854775808"
     },
     "latest_offsets_timestamp": "2024-10-14T21:26:07Z",
-    "phase": "PENDING",
+    "phase": "RUNNING",
```

Comment:
Quick question: Why did we change the status here? I thought it was `PENDING` on purpose to confirm that `waitForFlinkStatementToBeUpdated` is capable of sending multiple API requests.

Reply:
In a resume, before this PR, I put `PENDING` as target status, which means a PENDING statement will be considered as a successful resume. 

```
pendingStates = []string{stateStopped, stateResuming}
targetStates = []string{statePending, stateRunning, stateCompleted}
```

But in reality I have found that we can have many scenarios, where the statement can be stuck in `PENDING` state, in this case we should not give a less accurate signal user that resume has been completed, that's the reason I make this change:

```
pendingStates = []string{stateStopped, statePending}
targetStates = []string{stateRunning, stateCompleted}
```

Reply:
What I am trying to do is, create another json response to represent the `PENDING` state after a resume request, please stay tuned.